docs/core-components/OpenAPI-Specs.yaml

```
@@ -0,0 +1,533 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /auth/session:
+    get:
+      tags: [Authentication]
+      summary: Get current session
+      description: |
+        Get current authentication & authorization context.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)